wagtail/wagtailadmin/templates/wagtailadmin/shared/breadcrumb.html

```diff
@@ -1,8 +1,10 @@
 {% load i18n wagtailadmin_tags %}
 
 <ul class="breadcrumb">
-    {% for page in page.get_ancestors %}
-        {% if page.is_root %}
+    {% explorable_ancestors page request.user as ancestors %}
+    {% for page in ancestors %}
+        {% is_explorable_root page request.user as page_is_explorable_root %}
+        {% if page_is_explorable_root %}
```

Comment:
Yup, bug. I completely flubbed `Page.get_explorable_ancestors(user)`. The correct way to do it is much easier than what I came up with, too.

None of the tests in the existing test suite managed to catch this problem, which is how I managed to miss this bug. I'll remedy that.